platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/format/LineBasedFormat.java

```diff
@@ -24,33 +27,33 @@
 import com.swirlds.logging.api.extensions.event.LogMessage;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
-import java.time.Instant;
-import java.time.ZoneId;
-import java.time.format.DateTimeFormatter;
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * A utility class that formats a {@link LogEvent} as a line based format.
  */
 public class LineBasedFormat {
 
+    private static final String THREAD_SUFFIX = "UNDEFINED-THREAD";
+    private static final String LOGGER_SUFFIX = "UNDEFINED-LOGGER";
+    private static final String UNDEFINED_MESSAGE = "UNDEFINED-MESSAGE";
+    private static final String BROKEN_MESSAGE = "BROKEN-MESSAGE";
     /**
      * The emergency logger.
      */
     private static final EmergencyLogger EMERGENCY_LOGGER = EmergencyLoggerProvider.getEmergencyLogger();
 
-    /**
-     * The formatter for the timestamp.
-     */
-    private static final DateTimeFormatter FORMATTER =
-            DateTimeFormatter.ISO_LOCAL_DATE_TIME.withZone(ZoneId.systemDefault());
+    private final boolean formatTimestamp;
+
+    public LineBasedFormat(boolean formatTimestamp) {
```

Comment:
javadoc

Reply:
added + renamed the class for better stating its purpose